packages/sirius-web/backend/sirius-web-infrastructure/src/main/java/org/eclipse/sirius/web/infrastructure/configuration/i18n/I18nConfiguration.java

```diff
@@ -29,13 +32,14 @@ public class I18nConfiguration {
     public ResourceBundleMessageSource resourceBundleMessageSource() {
         var messageSource = new ResourceBundleMessageSource();
         messageSource.setBasenames("i18n/messages");
+        messageSource.setDefaultLocale(Locale.ENGLISH);
         messageSource.setDefaultEncoding(null);
         messageSource.setUseCodeAsDefaultMessage(true);
         return messageSource;
     }
 
     @Bean
-    public MessageSourceAccessor messageSourceAccessor(ResourceBundleMessageSource resourceBundleMessageSource) {
-        return new MessageSourceAccessor(resourceBundleMessageSource);
+    public MessageSourceAccessor messageSourceAccessor(ResourceBundleMessageSource resourceBundleMessageSource, @Value("${spring.mvc.locale:en_EN}") Locale locale) {
+        return new MessageSourceAccessor(resourceBundleMessageSource, locale);
     }
```

Comment:
Explanation for this change:
By default, the locale used by Spring is either (1) the one associated with the current thread or (2) the system's locale.
On my machine (configured in French), I observed that backend texts were always sent to the frontend in French (the system locale), regardless of whether I set `SPRING_MVC_LOCALE` to `fr_FR` or `en_EN`.
This change allows us to explicitly use the value defined in `SPRING_MVC_LOCALE`, or fall back to English if it is not defined, thereby avoiding system-dependent behavior.